metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1967,6 +1970,70 @@ class MetalsLanguageServer(
       case ServerCommands.NewScalaProject() =>
         newProjectProvider.createNewProjectFromTemplate().asJavaObject
 
+      case ServerCommands.SetupNotebookKernelForThisProject() =>
+        scribe.info("Executing SetupNotebookKernelForThisProject")
+        val source = sourceMapper.workspace()
+        scribe.info(source.toString)
+        val targetClasspath = buildTargets
+          .inferBuildTarget(source)
+          .flatMap(buildTargets.targetJarClasspath)
+          .getOrElse(Nil)
+          .map(_.toString)
+
+        scribe.info("Found classpath ")
+        scribe.info("targetClasspath " + targetClasspath.mkString("\n"))
+        val jvmReprRepo = coursierapi.MavenRepository.of(
+          "https://maven.imagej.net/content/repositories/public/"
```

Comment:
Why do you need the additional repo?

Reply:
Almond requires an artefact jvmrepr. 

The jvm repr artefact, is only available on jitpack, as far as I can tell.